PhotoGether/PresentationLayer/EditPhotoRoomFeature/EditPhotoRoomFeature/Source/EditPhotoRoomHostViewModel.swift

```diff
@@ -12,17 +12,15 @@ public final class EditPhotoRoomHostViewModel {
     }
     
     enum Output {
-        case emojiEntity(entity: EmojiEntity)
         case stickerObjectList([StickerEntity])
         case frameImage(image: UIImage)
+        case stickerBottomSheetPresent
```

Comment:
```suggestion
        case presentStickerBottomSheet
        // case shouldShowStickerBottomSheet
```
동사를 앞으로보내 스티커 바텀시트를 보여줘야한다. 라는 내용이 되면 좋을 것 같아요!

Reply:
[47e5553](https://github.com/boostcampwm-2024/iOS04-PhotoGether/pull/98/commits/47e55532e98e412c8973a6eee8abfd7c67bcd9d0)

좋은것 같아요👍

수정 완료했습니다!